opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/QueryQuery.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.opensearch.storage.script.filter.lucene.relevance;
+
+import com.google.common.collect.ImmutableMap;
+import org.opensearch.index.query.QueryBuilders;
+import org.opensearch.index.query.QueryStringQueryBuilder;
+
+/**
+ * Class for Lucene query that builds the query_string query.
+ */
+public class QueryQuery extends NoFieldQuery {
```

Comment:
```suggestion
public class QueryQuery extends NoFieldQuery<QueryStringQueryBuilder> {
```

Reply:
Fixed in 9d1497503f73f2baaba63c7e637b712a92ed214c